### DISCUSSION TITLE
Skip missing directory when walking through history

### DISCUSSION THREAD

**Discussion by GnuT0203** [doublecmd/doublecmd#1439] [Suggestions]
Hi
When I go back (or forth) through the directory history and come to a deleted directory i get an error message. That's fine but from this point I cannot go on. Double Commander retries to enter that missing directory and fails. I would suggest to skip missing directories. 
For comparison: Total Commander shows an error message too but on the next attempt it skips that directory.

**Comment by ciros88** [comment]
in my everyday use, for various reasons, i prefer to see the error instead of skipping them.
but, at least, total commander behavior might be acceptable also.

**Reply by GnuT0203** [reply]
Yes, agree. Seeing the error message is a benefit to get to know that, oops, someone removed that directory. But from that point I am stalled. DC continues retrying to enter the missing directory, I cannot go on further in the history. My idea is to skip that directory after first attempt to enter it and the error message has been shown.
